_posts/2024-02-24-ruesselsheim-trip.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Ausflug nach Rüsselsheim"
+image: /assets/images/posts/24_ruesselsheim/rue_spring.jpg
+thumbnail: /assets/images/posts/thumbnails/rue_spring_thumb.jpg
+tags: events
+author: moritz
+---
+
+Am 24. Februar begab sich das Discgolf Syndikat zu ein paar Runden Discgolf nach Rüsselsheim. Uns lockte der abwechslungsreiche und anspruchsvolle Kurs im [Ostpark](https://udisc.com/courses/ruesselsheim-ostpark-wYvu).
+
+### Anfahrt nach Rüsselsheim
+
+Die Anfahrt ging schneller vorbei als gedacht. 
+Nachdem wir ein und für alle mal geklärt haben welche Kartoffeln man am besten für Kartoffelsalat benutzen sollten, waren wir dann auch schon am Ziel angekommen.
+Obwohl die Autofahrt noch durch einige Regenschauer getrübt war, schien uns das Wetter wohlgesonnen zu sein, da sich die Wolkendecke öffnete und die Sonne uns mit ihrer Anwesenheit beglückte.
+Auch hier konnten man spüren, dass es so langsam Frühling wird.
+
+Somit waren wir um 10 Uhr startbereit am Parcours und konnten mit der ersten Runde starten.
+Hier haben uns auch ein paar Spieler*innen des ansässigen [Scheibensucher e.V.](http://www.scheibensucher.de/wp/) begrüßt.
+
+### Wurf für Wurf
+
+![Kerstin am Abwurf](/assets/images/posts/24_ruesselsheim/rue_wurf.jpeg)
+
+Der Discgolf-Parcours in Rüsselsheim erwies sich als echte Herausforderung mit seinen 24 abwechslungsreichen Bahnen. Von offenen Wiesenflächen bis hin zu dichten Wäldern bot jeder Abschnitt eine neue und aufregende Spielsituation.
+Auch wenn die Line nicht immer klar war, mit gegenseitigen Tipps und gelegentlichen Tricks ging es von Korb zu Korb.
+
+![Beratung zur besten Line ist nicht immer einfach](/assets/images/posts/24_ruesselsheim/rue_hands_up.jpeg)
+
+
+Außerdem war auch stets eine gesunde Portion Glück mit dabei.
+
+![Korb erreicht...egal wie!](/assets/images/posts/24_ruesselsheim/rue_close.jpg)
+
+Trotz der teils herausfordernden Bahnen, ließen wir uns nicht abschrecken und haben es geschafft zwei vollen Runden zu spielen.
+
+![Moritz gibt alles](/assets/images/posts/24_ruesselsheim/rue_full_spin.jpeg)
+
+Als die letzten Bahnen gespielt waren und die Sonne langsam hinter dem Horizont verschwand, kehrten wir, nach einem kleinen Snack zur Stärkung, erschöpft, aber zufrieden nach Köln zurück.
+
+![Abfahrt nach Köln](/assets/images/posts/24_ruesselsheim/rue_abfahrt.jpeg)
+
+Alles in allem war es ein super Tag, um einen neuen Discgolf-Parcours zu entdecken und einfach ein bisschen Spaß auf und abseits des Parcours zu haben. Wenn ihr auch Lust habt an zukünftigen Ausflügen teilzunehmen, haltet einfach in unserem Discord ([link](https://discord.com/invite/bus8ZcaNFT)) danach Ausschau. 
```

Comment:
haltet einfach [in unserem Discord](https://discord.com/invite/bus8ZcaNFT) danach Ausschau